### DISCUSSION TITLE
Missing basic information

### DISCUSSION THREAD

**Discussion by ghost** [npm/feedback#196] [Website]
On https://www.npmjs.com/ you cannot find the current version of npm and no obvious download or install button. Its just a package search and promotes pricing and pro. It does not provide most basic information about npm itself. Most prominent is signing up (without explaining what you sign up for) and learn about pro. The site is practically useless - except for the package search. There isnt even a prominent link to the github repo of npm. I think the way this website is made now is completely misleading.

**Comment by MylesBorins** [comment]
Hey @einicher
 -
From the main site you can click Documentation -> Getting Started -> Configuring your local environment -> downloading and install node.js and npm

https://docs.npmjs.com/downloading-and-installing-node-js-and-npm

I'll admit that we are definitely burying the lede on how to get started and perhaps could surface that far sooner in the experience.
